sdk/resourcemanager/azure-resourcemanager-servicebus/src/main/java/com/azure/resourcemanager/servicebus/implementation/TopicAuthorizationRuleImpl.java

```diff
@@ -62,7 +62,7 @@ protected Mono<TopicAuthorizationRule> createChildResourceAsync() {
                 this.namespaceName(),
                 this.topicName(),
                 this.name(),
-                this.innerModel().rights())
+                this.innerModel())
```

Comment:
what causes this change?

Reply:
I believe it's m4. Paramter.isFlattened is false this time.